src/test/java/com/google/api/codegen/testdata/discoveries/sheets.v4.json.overrides

```diff
@@ -1,12 +1,267 @@
 {
   "java": {
     "methods": {
+      "sheets.spreadsheets.batchUpdate": {
+        "requestBodyType": {
```

Comment:
This is true... I had considered both originally, and I am not opposed to doing so, but I figured I'd do the easy way first since it requires no extra work. Since no other APIs have asked yet, are you okay with deferring what you're suggesting for some time?

The method you're suggesting is an extension on what we're doing here that requires recursively exploring both non-required parameters and request body types, neither of which are implemented in `DiscoveryImporter`/`ApiaryConfigToSampleConfigConverter`. I can open a bug to track that feature, I wanted to do it eventually but haven't gotten to it.